Eradication/VectorPopulationIndividual.cpp

```diff
@@ -847,13 +846,18 @@ namespace Kernel
         {
             throw QueryInterfaceException( __FILE__, __LINE__, __FUNCTION__, "m_context->GetParent()", "IVectorSimulationContext", "ISimulationContext" );
         }
-        suids::suid current_node = m_context->GetSuid();
+        VectorGender::Enum     vector_gender = VectorGender::VECTOR_FEMALE;
+        Gender::Enum human_gender_equivalent = m_pMigrationInfoVector->ConvertVectorGender( vector_gender );
```

Comment:
By putting stuff into GetFractionTraveling(), this code doesn't even have to know about the human vs vector gender stuff.